addons/website_sale/static/src/js/website_sale.js

```diff
@@ -1197,7 +1197,9 @@ publicWidget.registry.multirangePriceSelector = publicWidget.Widget.extend({
         if (parseFloat(range.max) !== range.valueHigh) {
             search['max_price'] = range.valueHigh;
         }
-        this.el.querySelector('.o_wsale_products_grid_table_wrapper').classList.add('opacity-50');
+        let product_list_div = this.el.querySelector('.o_wsale_products_grid_table_wrapper');
+        if (product_list_div) { product_list_div.classList.add('opacity-50');}
```

Comment:
I think at least JS framework prefers:

```suggestion
        if (product_list_div) {
            product_list_div.classList.add('opacity-50');
        }
```